plugin.xml

```diff
@@ -99,21 +99,21 @@
     <framework src="CoreTelephony.framework"/>
     <framework src="EventKit.framework"/>
     <framework src="EventKitUI.framework"/>
+    <framework src="MediaPlayer.framework"/>
     <framework src="MobileCoreServices.framework"/>
     <framework src="QuartzCore.framework"/>
     <framework src="Security.framework"/>
     <framework src="Social.framework"/>
     <framework src="StoreKit.framework"/>
+    <framework src="SystemConfiguration.framework"/>
     <framework src="iAd.framework"/>
 
     <framework src="libxml2.dylib"/>
+    <framework src="libsqlite3.dylib" /> 
```

Comment:
@MaxGabriel might know if they're still around in El Capitan, I think he had issues. Either way, we'll be able to figure this out later, it's an issue with the Unity SDK also for the same reasons